CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.16.0]
+### Changed
+- **BREAKING:** Refactor code to split keyring from bridge logic. ([#156](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/156))
+- Migrate code to TypeScript. ([#174](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/174))
```

Comment:
This should probably be two entries: one for the added types (non-breaking), and one for the change from default export to named export (breaking).

Anything else worth noting here @mikesposito ?

Reply:
Done.